pyramid-app/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+def includeme(_config):  # pragma: no cover
+    pass
```

Comment:
This is needed for `make_db` to work: there's a `config.include("models")` that needs to work